site_root/source/resources/StructureDefinition-age-at-event.json

```diff
@@ -0,0 +1,67 @@
+{
+    "resourceType":"StructureDefinition",
+    "id":"age-at-event",
+    "meta":{
+        "lastUpdated":"2019-12-20T21:25:21.775+00:00"
+    },
+    "url":"http://fhir.kids-first.io/StructureDefinition/age-at-event",
+    "version":"0.1.0",
+    "name":"age-at-event",
+    "title":"Age At Event",
+    "status":"draft",
+    "description":"Age (in days since birth) of the Participant at time of event",
+    "fhirVersion":"4.0.1",
+    "kind":"complex-type",
+    "abstract":false,
+    "context": [
+        {
+            "type":"element",
+            "expression":"Observation"
+        },
+        {
+            "type":"element",
+            "expression":"Condition"
+        }
+    ],
+    "type":"Extension",
+    "baseDefinition":"http://hl7.org/fhir/StructureDefinition/Extension",
+    "derivation":"constraint",
+    "differential":{
+        "element": [
+            {
+                "id":"Extension",
+                "path":"Extension",
+                "short":"Age at event",
+                "definition":"Age (in days since birth) of the Participant at time of event",
+                "min": 0,
+                "max":"1",
+                "mustSupport":true,
+                "isModifier":false
+            },
+            {
+                "id":"Extension.extension",
+                "path":"Extension.extension",
+                "max":"0"
```

Comment:
This isn't really needed, and in general any data elements should be extensible.

Reply:
@ShahimEssaid  Ok I wasn't sure. I think I copied this from somewhere else.